dependencies.gradle

```diff
@@ -1,117 +1,113 @@
-
-        supportAnnotations      : "com.android.support:support-annotations:$versions.supportLibs",
-        supportAppCompat        : "com.android.support:appcompat-v7:$versions.supportLibs",
-        supportDesign           : "com.android.support:design:$versions.supportLibs",
-        supportRecyclerView     : "com.android.support:recyclerview-v7:$versions.supportLibs",
-        supportCardView         : "com.android.support:cardview-v7:$versions.supportLibs",
-
-        butterKnife             : "com.jakewharton:butterknife:$versions.butterKnife",
-        butterKnifeCompiler     : "com.jakewharton:butterknife-compiler:$versions.butterKnife",
-        picasso                 : "com.squareup.picasso:picasso:$versions.picasso",
-        picassoOkHttpDownloader : "com.jakewharton.picasso:picasso2-okhttp3-downloader:$versions.picassoOkHttpDownloader",
-        timber                  : "com.jakewharton.timber:timber:$versions.timber",
-
-        // Analytics https://tech.yandex.com/metrica-mobile-sdk/
-        yandexAppMetrica        : "com.yandex.android:mobmetricalib:$versions.yandexAppMetrica",
-
-        // Developer Tools
-        stetho                  : "com.facebook.stetho:stetho:$versions.stetho",
-        stethoOkHttp            : "com.facebook.stetho:stetho-okhttp3:$versions.stetho",
-        leakCanary              : "com.squareup.leakcanary:leakcanary-android:$versions.leakCanary",
-        tinyDancer              : "com.github.brianPlummer:tinydancer:$versions.tinyDancer",
-        paperwork               : "hu.supercluster:paperwork:$versions.paperworkGradlePlugin",
-        lynx                    : "com.github.pedrovgs:lynx:$versions.lynx",
-        devMetricsNoOp          : "com.frogermcs.androiddevmetrics:androiddevmetrics-runtime-noop:$versions.androidDevMetricsGradlePlugin",
-        processPhoenix          : "com.jakewharton:process-phoenix:$versions.processPhoenix",
-
-        // Test dependencies
-        junit                   : "junit:junit:$versions.junit",
-        robolectric             : "org.robolectric:robolectric:$versions.robolectric",
-        assertJ                 : "org.assertj:assertj-core:$versions.assertJ",
-        equalsVerifier          : "nl.jqno.equalsverifier:equalsverifier:$versions.equalsVerifier",
-        okHttpMockWebServer     : "com.squareup.okhttp3:mockwebserver:$versions.okHttp",
-        mockitoCore             : "org.mockito:mockito-core:$versions.mockito",
-        supportTestRunner       : "com.android.support.test:runner:$versions.supportTestRunner",
-        supportTestRules        : "com.android.support.test:rules:$versions.supportTestRunner",
-        espressoCore            : "com.android.support.test.espresso:espresso-core:$versions.espresso",
-        espressoContrib         : "com.android.support.test.espresso:espresso-contrib:$versions.espresso",
-]
+ext.versions = [code                         : 1,
```

Comment:
@artem-zinnatullin append `?=w1` to the link and it'll filter out diffs. [magic](https://github.com/artem-zinnatullin/qualitymatters/pull/202/files?w=1)